### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by janden**
Depends on #687.

### CODE REVIEW

**Comment by janden** [comment]
It's not pushing to GH pages yet. Need to reverse engineer what @eugenium did for Circle CI.

**Comment by eickenberg** [comment]
> It's not pushing to GH pages yet. Need to reverse engineer what @eugenium did for Circle CI.

It is possible that I was also involved in that. I'll try to take a look later

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                              -f https://download.pytorch.org/whl/torch_stable.html
+            - name: Set up Kymatio
+              run: python3 setup.py develop
+            - name: Generate documentation
+              run: |
+                  pushd doc
+                  make clean
+                  #make html
+                  mkdir -p build/html
+                  echo "haha" > build/html/index.html
```
Comment (eickenberg) [inline .github/workflows/docs.yml:38]: haha :rofl:

Comment (janden) [reply .github/workflows/docs.yml:38]: Well, you gotta be able to laugh a little sometimes…

**Pull request closed without merging**